Source/Krypton Components/Directory.Build.targets

```diff
@@ -156,8 +156,7 @@
 				<Description Condition="'$(TFMs)' == 'all'">
 					An update to Component factory's krypton toolkit to support .NET Framework 4.6.2 - 4.8.1, .NET Core 3.1 and .NET 6 - 7. $(Description)
 					This package supports all .NET Framework versions starting .NET Framework 4.6.2 - 4.8.1, .NET Core 3.1 and .NET 6 - 7.
-					Also, all libraries are included targeting each specific framework version for performance purposes.
-					To view all of the standard toolkit package latest version information, please visit: https://github.com/Krypton-Suite/Krypton-Toolkit-Suite-Version-Dashboard/blob/main/Documents/Modules/Standard/Krypton-Toolkit-Suite-Standard-Modules.md
+					Also, all libraries are included targeting each specific framework version for performance purposes. To view all of the standard toolkit package latest version information, please visit: https://github.com/Krypton-Suite/Krypton-Toolkit-Suite-Version-Dashboard/blob/main/Documents/Modules/Standard/Krypton-Toolkit-Suite-Standard-Modules.md
```

Comment:
Nope not done, Spacing still exists

Reply:
> Nope not done, Spacing still exists

Is it the layout of GitHub, as it's now all on one line?

Reply:
Seems to work now?

![image](https://user-images.githubusercontent.com/949607/208722824-10fbe849-0481-4d1d-b79e-a425165e18df.png)